app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -256,7 +254,9 @@ const SendBitcoinDetailsScreen = ({
   const [isModalVisible, setIsModalVisible] = useState(false)
   const [errorMessage, setErrorMessage] = useState("")
   const [validAmount, setValidAmount] = useState(false)
-  const usdDisabled = paymentType === "onchain" || usdWalletId === undefined
+  const usdDisabled =
```

Comment:
I feel we could remove this assumption for now. I don't think we need this complexity of managing having or not having usd enabled. can be done as part of another PR.